icons/threema.svg

```diff
@@ -1 +1 @@
-<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Threema icon</title><path d="M13.755 22.243a1.756 1.756 0 11-3.512.002 1.756 1.756 0 013.512-.002zm-6.335 0a1.756 1.756 0 11-3.513.002 1.756 1.756 0 013.513-.002zm12.671 0a1.756 1.756 0 11-3.512.002 1.756 1.756 0 013.512-.002zM7.477 17.248l-5.171 1.293 1.105-4.42c-1.094-1.442-1.733-3.175-1.733-5.039C1.678 4.066 6.3 0 12 0c5.7 0 10.322 4.066 10.322 9.082 0 5.016-4.622 9.083-10.322 9.083a11.45 11.45 0 01-4.523-.917zm1.595-8.92h-.115a.47.47 0 00-.47.471v4.126c0 .26.21.471.47.471h6.086c.26 0 .47-.21.47-.47V8.798a.47.47 0 00-.47-.47h-.115v-1.17a2.927 2.927 0 00-2.93-2.924A2.926 2.926 0 009.073 7.16zm4.685 0h-3.514V7.16a1.756 1.756 0 013.514 0v1.17Z"/></svg>
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Threema icon</title><path d="M13.75496 22.24242a1.75596 1.75596 0 11-3.51192.002 1.75596 1.75596 0 013.51192-.002zm-6.33486 0a1.75646 1.75646 0 11-3.51293.002 1.75646 1.75646 0 013.51293-.002zm12.67073 0a1.75596 1.75596 0 11-3.51193.002 1.75596 1.75596 0 013.51193-.002zM7.4771 17.24752L2.3062 18.5405l1.10498-4.41991c-1.09398-1.44197-1.73296-3.17493-1.73296-5.0389C1.67822 4.0658 6.30011-.0001 12-.0001s10.32179 4.0659 10.32179 9.0818c0 5.0159-4.6219 9.08282-10.32179 9.08282a11.44976 11.44976 0 01-4.5229-.91699zm1.59497-8.91981h-.115a.46999.46999 0 00-.46999.471v4.1259c0 .26.21.471.47.471h6.08587c.26 0 .46999-.21.46999-.47v-4.1279a.46999.46999 0 00-.47-.47h-.115V7.15773A2.92694 2.92694 0 0011.998 4.2338a2.92594 2.92594 0 00-2.92494 2.92593zm4.6849 0h-3.51392V7.15973a1.75696 1.75696 0 013.51392 0v1.16998z"/></svg>
```

Comment:
LGTM! Any chance you can get it down to 3 decimals?

Reply:
@adamrusted Done 🔒